Frontend/ui-customer/src/Components/MessagingView/View.purs

```diff
@@ -0,0 +1,515 @@
+  , width MATCH_PARENT
+  , accessibility DISABLE
+  ][]
+
+suggestionsView :: forall w. Config -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
+suggestionsView config push =
+  linearLayout
+  [ height WRAP_CONTENT
+  , width MATCH_PARENT
+  , afterRender push $ const $ NoAction
+  , visibility if (length config.suggestionsList == 0 ) || not config.canSendSuggestion || not config.enableSuggestions then GONE else VISIBLE
+  , background Color.white900
+  , accessibility DISABLE
+  , margin $ MarginBottom 16
+  ][ textView $
+     [ text $ getString QUICK <> "\n" <> getString REPLIES
+     , color Color.black700
+     , margin $ MarginRight 12
+     , accessibility ENABLE
+     , accessibilityHint $ "Quick Replies"
+     ] <> FontStyle.captions TypoGraphy
+   , horizontalScrollView
+     [ height WRAP_CONTENT
+     , weight 1.0
+     , scrollBarX false
+     , accessibility DISABLE
+     , disableKeyboardAvoidance true
+     ][ linearLayout
+        [ height WRAP_CONTENT
+        , width WRAP_CONTENT
+        , gravity LEFT
+        , accessibility DISABLE
+        ] (mapWithIndex (\index item -> quickMessageView config item index push)(config.suggestionsList))
+      ]
+  ]
+
+dummyTextView :: forall w . PrestoDOM (Effect Unit) w
+dummyTextView =
+  textView
+  [ width WRAP_CONTENT
+  , height WRAP_CONTENT
+  ]
+
+quickMessageView :: forall w. Config -> String -> Int -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
+quickMessageView config message idx push =
+  let value = getMessageFromKey message config.languageKey
+  in
+  linearLayout
+  [ height WRAP_CONTENT
+  , width $ WRAP_CONTENT
+  , onClick push $ const $ SendSuggestion message
+  , background Color.blue600
+  , visibility $ if value /= "" then VISIBLE else GONE
+  , cornerRadius 12.0
+  , accessibility ENABLE
+  , accessibilityHint $ (getMessageFromKey message "EN_US") <> " : Button : Select to send message to driver"
+  , margin $ MarginLeft if idx == 0 then 0 else 8
+  , padding $ Padding 16 6 16 6
+  ][ textView $
+     [ text $ value
+     , accessibility DISABLE
+     , color Color.black900
+     ] <> FontStyle.tags TypoGraphy
+  ]
+chatComponent :: forall w. Config -> (Action -> Effect Unit) -> ChatComponent -> Boolean -> String -> Int -> PrestoDOM (Effect Unit) w
+chatComponent state push config isLastItem userType index =
+  let message = getMessageFromKey config.message state.languageKey
+  in
+  PrestoAnim.animationSet
+    [ if state.userConfig.appType == config.sentBy then
+         translateInXForwardAnim $ if isLastItem then true else false
+      else
+          translateInXBackwardAnim $ if isLastItem then true else false
+    ]
+  $ linearLayout
+  [height WRAP_CONTENT
+  , width MATCH_PARENT
+  , accessibility ENABLE
+  , accessibilityHint $ (if config.sentBy == "Customer" then "You Sent : " else "Driver Sent : ") <> getMessageFromKey config.message "EN_US"
+  , margin (getChatConfig state config.sentBy isLastItem index).margin
```

Comment:
`getChatConfig state config.sentBy isLastItem index`
This is being called multiple times. Better to store it as `chatConfig` and then use everywhere in this view